phpcs-sniffs/Formidable/Sniffs/Commenting/AddReturnVoidSniff.php

```
@@ -0,0 +1,281 @@
+
+	/**
+	 * Find the position to insert @return void.
+	 *
+	 * @param File $phpcsFile  The file being scanned.
+	 * @param int  $docStart   The docblock opener position.
+	 * @param int  $docEnd     The docblock closer position.
+	 *
+	 * @return false|int The position to insert after, or false if not found.
+	 */
+	private function findInsertPosition( File $phpcsFile, $docStart, $docEnd ) {
+		$tokens = $phpcsFile->getTokens();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$tokens'. (undefined)

(UnusedLocalVariable)